crates/aptos/src/node/mod.rs

```diff
@@ -486,3 +494,174 @@ impl CliCommand<serde_json::Value> for ShowValidatorSet {
         Ok(response.into_inner())
     }
 }
+
+/// Tool for running a local testnet
+///
+#[derive(Parser)]
+pub enum LocalTestnetTool {
+    Node(RunTestNode),
+    MintCoins(MintCoins),
+}
+
+impl LocalTestnetTool {
+    pub async fn execute(self) -> CliResult {
+        use LocalTestnetTool::*;
+        match self {
+            Node(tool) => tool.execute_serialized().await,
+            MintCoins(tool) => tool.execute_serialized().await,
+        }
+    }
+}
+
+/// Mint coins to a set of accounts
+#[derive(Parser)]
+pub struct MintCoins {
+    /// Rest API server URL
+    #[clap(long, default_value = "http://localhost:8080")]
+    pub server_url: String,
+    /// Path to the private key for creating test account and minting coins.
+    /// To keep Testnet simple, we used one private key for aptos root account
+    /// To manually generate a keypair, use generate-key:
+    /// `cargo run -p generate-keypair -- -o <output_file_path>`
+    #[clap(long, default_value = "/opt/aptos/mint.key")]
+    pub mint_key_file_path: String,
```

Comment:
1. Can we make the path here the same path as the other command and use a constant for that? We might need to use the `const_format` crate.
2. We can validate the path right here with something like this, instead of using String:

```
#[clap(long, parse(from_os_str))]
pub mint_key_file_path: PathBuf
```

Reply:
I'm just going to nuke this PR